README.md

```diff
@@ -52,11 +48,11 @@ Development, and test the extension. :)
 To package the extension, you can do:
 
 ```bash
-npm install -g vsce
+sudo npm install -g vsce
```

Comment:
That's not a good idea. Never use `sudo` for development.

Let's figure out how to do this without `sudo`.

Reply:
We have to find another way.

Reply:
There is no need to install it globally, it works well locally:

```bash
npm install vsce
npx vsce package
```

Reply:
Yes, this works.